src/Avalonia.Styling/Styling/Styles.cs

```diff
@@ -38,7 +39,7 @@ public Styles()
                         ResourcesChanged?.Invoke(this, new ResourcesChangedEventArgs());
                     }
 
-                    x.ResourcesChanged += SubResourceChanged;
+                    x.ResourcesChanged += NotifyResourcesChanged;
```

Comment:
Not something that we want to fix within this PR but I wanted to ask if we have any other alternatives to using events for this? We create so many delegates for this that in calendar benchmark it was like the 6th top allocation.

![image](https://user-images.githubusercontent.com/2588062/75100500-7c953a80-55ce-11ea-9d7d-b360fc71e7ac.png)

![image](https://user-images.githubusercontent.com/2588062/75100506-9cc4f980-55ce-11ea-955a-d2c6538f524c.png)

Reply:
Yeah we could probably remove the usage of events for this.